ci/stage-test-sgx.jenkinsfile

```diff
@@ -264,6 +264,99 @@ stage('test-sgx') {
             env.build_ok = false
             sh 'echo "Rust example Test Failed"'
         }
+
```

Comment:
Same comment as above. Need for avoiding running the workloads under Cent and RHEL?